pkg/cmd/server/api/v1/conversions.go

```diff
@@ -131,6 +131,10 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
 			if len(obj.BindNetwork) == 0 {
 				obj.BindNetwork = "tcp4"
 			}
+			if obj.AllowRecursiveQueries == nil {
```

Comment:
Yes.  Admin will have to open the port if they want to make it open
for others, masters will default to 8053.